scripts/upload_model.py

```diff
@@ -41,6 +40,11 @@ def get_config():
         default=None,
         help="If provided, loads a previously trained HF model from the specified directory",
     )
+    parser.add_argument(
+        "--upload_b16",
+        action="store_true",  # Currently defaults to false. Flip post 7b block.
```

Comment:
WDYT about defaulting this to True? That _feels_ like the more desirable default

Reply:
Done.